docs/analytics/overview.mdx

```diff
@@ -10,38 +10,6 @@ Analytics reports help you understand clearly how your users behave, which enabl
 
 <YouTube id="8iZpH7O6zXo" />
 
-## Installation
-
-### 1. Make sure to initialize Firebase
-
-[Follow this guide](../overview.mdx) to install `firebase_core` and initialize Firebase if you haven't already.
-
-### 2. Add dependency
-
-On the root of your Flutter project, run the following command to install the plugin:
-
-```bash {5}
-flutter pub add firebase_analytics
-```
-
-### 3. Rebuild your app
-
-Once complete, rebuild your Flutter application:
-
-```bash
-flutter run
-```
-
-### 4. (optional) Apple [App Tracking Transparency](https://developer.apple.com/documentation/apptrackingtransparency)
-
-If you wish to use Firebase Analytics without IDFA collection capability, open your `/{ios|macos}/Podfile` and add the following global variable to the top of the file:
-
-```ruby
-$FirebaseAnalyticsWithoutAdIdSupport = true
-```
```

Comment:
Can we still cover this in the docs? Looks like it's removed completely

Context; Apple has a strict no usage of Ad Ids ("IDFA") in Kids Category apps - if your app has this and is for this category it will not pass review since they will not accept any app in the Kids category if the app accesses the IDFA iOS symbols.

Reply:
If it helps, it's a solution to provide support based on official iOS SDK docs here: https://firebase.google.com/docs/analytics/configure-data-collection?platform=ios#disable-IDFA-collection

Reply:
Done. Do we need the IDFV section too?

Reply:
> Done. Do we need the IDFV section too?

I'll make a separate PR for this (with docs) and add a top level Podfile variable too, since we can inject the Info.plist property internally if that variable is set